docs/resources/Voice.md

```diff
@@ -2,15 +2,15 @@
 
 ### Voice State Object
 
-Used to represent a user's voice connection status.
+The voice state object is used to represent a user's voice connection status.
```

Comment:
This is redundant

Reply:
This was one of my tries to fix these incomplete/fragment sentences; I will revert this because this would cause only inconsistency.